### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZuowenWang0000**
update a cvprw 24 paper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -618,8 +618,10 @@ IEEE Trans. Vis. Comput. Graphics (Proc. VR), 2021. [YouTube](https://youtu.be/-
 - <a name="Chen23biocas"></a>Chen, Q., Wang, Z., Liu, S.-C., Gao, C.,  
 *[3ET: Efficient Event-based Eye Tracking using a Change-Based ConvLSTM Network](https://arxiv.org/abs/2308.11771)*,  
 IEEE BioCAS Conf., 2023. [YouTube](https://www.youtube.com/watch?v=aRB5mDNfrHM), [Code](https://github.com/qinche106/cb-convlstm-eyetracking)
-
-
+- <a name="Wang24cvprw"></a>Wang, Z., Gao, C., Wu, Z., Conde, M., Timofte, R., Liu, S.-C., Chen, Q., et al.
```
Comment (guillermogb) [inline README.md:621]: could you add two blank spaces at the end of author list and at the end of the title so that the entry spans 3 lines?

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Review by ZuowenWang0000 — COMMENTED** [review]
added spaces after author list to start a new line

**Review by guillermogb — APPROVED** [review]
thanks

**Comment by ZuowenWang0000** [comment]
> thanks

thanks for maintaining the repo!

**Pull request merged**